plugin/xule/XuleModelIndexer.py

```diff
@@ -258,8 +259,26 @@ def get_decimalized_value(fact_a, fact_b, xule_context):
 
     min_decimals = min(fact_a_decimals, fact_b_decimals)
 
-    fact_a_value = fact_a.xValue if fact_a_decimals == float('inf') else round(fact_a.xValue, min_decimals)
-    fact_b_value = fact_b.xValue if fact_b_decimals == float('inf') else round(fact_b.xValue, min_decimals)
+    def _safe_round_fact_value(fact, min_decimals):
+        if min_decimals == float('inf'):
```

Comment:
should 

if **min_decimals == float('inf'):**

be 

if **get_decimals(fact, xule_context) == float('inf'):**

instead ??